app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>Here is all user history information</h1>
\ No newline at end of file
+<h1>Here is all <%= @user.name %> history information</h1>
+
+<div class="users-container">
+  <img class="user-photo" src="<%= @user.photo %>"><br>
+  <h2><%= @user.name %></h2><br>
+  <p class="text-usercard-index">Number of Posts: <%= @user.posts_counter %></p><br><br>
+</div>
+<div class="bio-container">
+  <p class="text-bio">
+   Bio:<br><br>
+   <%= @user.bio %>
+  </p>
+</div>
+<% @user.posts.each_with_index do |post, index| %>
+<div class="posts-container">
+  <div class="posts-title-container">
+  Post #<%= index + 1 %>:<br>
+  <p class="text-post"><%= post.text %></p><br>
+  </div>
+  <p class="post-footer-text">comments:<%= post.comments_counter %> likes:<%= post.likes_counter %></p><br><br>
+</div>
+  <% end %><br><br>
+<div class="see-all-posts-btn"><button>See all Posts</button></div>
```

Comment:
- [ ] Still in line with the previous review. This should also be a link that directs a user to all posts. Please add the link.